### PR TITLE
recipes/google_ner_pre_labeling: fix by adapting to up-to-date Kili api

### PR DESCRIPTION
**Pull request by PierreLeveau**
This PR updates the `recipes/google_ner_pre_labeling.py` to use the up-to-date kili api. It also add a number of features like:
* batch import of assets and predictions.
* creation of the project from scratch to better control the named entity types.
* removal of the asset prioritization (I can put it back if needed)
* formatting / clean-up

### CODE REVIEW

File: recipes/google_ner_pre_labeling.py [hunk 1]
```diff
@@ -1,90 +1,188 @@
...
+
+    # Insert assets
+    print("Import asset list into Kili")
+    kili.append_many_to_dataset(
+        project_id=project_id,
+        content_array=[a["escaped_text"] for a in assets],
+        external_id_array=[a["external_id"] for a in assets]
+    )
+
+    # Add predictions
```
Comment (marcenacp) [inline recipes/google_ner_pre_labeling.py:181]: Useless comment

File: recipes/README.md [hunk 2]
```diff
@@ -49,16 +49,26 @@ Check out [our recipe on how to export labels](https://github.com/kili-technolog
 
 ## How to do Named-Entity Recognition with Google
 
-Example based on Enron email dataset. Its downloads the data, get the first 50 emails, pre-label them with Google NLP, push both assets and predictions to Kili and prioritize the assets.
-
-1. Create a NER project and retrieve its ID.
-
-2. Make sure that your GCP authentication is set up properly (or follow [this tutorial](https://cloud.google.com/natural-language/docs/reference/libraries)).
+Example based on Enron email dataset. Its performs the following:
```
Comment (marcenacp) [inline recipes/README.md:52]: Install a Markdown formatter on VSCode

File: recipes/README.md [hunk 3]
```diff
@@ -49,16 +49,26 @@ Check out [our recipe on how to export labels](https://github.com/kili-technolog
...
+  * downloads the data.
```
Comment (marcenacp) [inline recipes/README.md:53]: For commit names please read: https://www.notion.so/Conventional-Commits-d11e2f19f503440ebd300d1893dae2bf

**Comment by PierreLeveau** [comment]
changes done

**Review by theodu — APPROVED** [review]

**Review by p-desaintchamas — APPROVED** [review]

**Pull request merged**